runtime/oti/j9nonbuilder.h

```diff
@@ -1849,7 +1849,7 @@ typedef struct J9AnnotationState {
 } J9AnnotationState;
 
 typedef struct J9Module {
-	j9object_t moduleName;
+	struct J9UTF8* moduleName;
```

Comment:
Space before, not after, `*`.

Reply:
I understand that right-aligned `*` for pointer types is the more commonly enforced rule, but all the pointer types in this `struct` and other `struct`s close to this one, use left-aligned `*`. If we want them to all be right-aligned, I suggest we change them all at once in a separate PR.

Reply:
In such cases, the recommended approach is to follow the existing formatting in the file: [ref](https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#old-code).

Reply:
I don't think of it as a "rule", but a way of writing code that's less misleading for human readers.

If we defer every suggested improvement to a later pull request, we will make progress more slowly. I was only asking that this field (the line being changed) be expressed properly; but if you want to fix the remaining fields of this structure, that would be great.